### ISSUE TITLE
[BUG] Shopping list does not include beacons or modules in beacons

### ISSUE THREAD

**Issue by wesdotcool** [ShadowTheAge/yafc#137]
YAFC 0.5.8.0

The beacons and their modules are completely ignored in this list 
![image](https://user-images.githubusercontent.com/246963/133485145-9f40c417-be24-4eae-b1d4-f2b2caead6e7.png)

**Comment by ShadowTheAge** [comment]
yes and that is impossible to implement because there is no way to figure out your beacon layout

**Comment by wesdotcool** [comment]
Could there be a checkbox on this screen to include beacons. Instead of figuring out the beacon layout, it would simply add up all of the beacons and their modules giving you the maximum amount of beacons.

The number of beacons in certain builds can add up really fast, so it would be nice to have a total somewhere of how many beacons could possibly be used. It would also help estimate final energy usage too

**Comment by ShadowTheAge** [comment]
I don't think that number will be any closer to real number than the current (zero)

**Comment by ShadowTheAge** [comment]
What I think I can do is add "beacon sharing coefficient" to preferences and when and only when it is set beacons will appear to the shopping list
